packages/features/bookings/lib/handleCancelBooking.ts

```diff
@@ -496,11 +504,9 @@ async function handler(req: CustomRequest) {
 
   // If the video reference has a credentialId find the specific credential
   if (bookingVideoReference && bookingVideoReference.credentialId) {
-    const { credentialId, uid } = bookingVideoReference;
-    if (credentialId) {
-      const videoCredential = bookingToDelete.user.credentials.find(
-        (credential) => credential.id === credentialId
-      );
+    const { credentialId, uid, type } = bookingVideoReference;
```

Comment:
I reverted this entire file and was still able to cancel the webex meeting by cancelling the booking

Reply:
@hariombalhara 

the cancel booking behavior is not consistent for me. I reverted the file to the original code and am running into errors again while canceling the booking. The `videoAdapter` is not picked correctly from credentials. This results in the meeting getting deleted on cal, but not on Webex. I added logs in a couple of places to show this.

<img width="693" alt="Screenshot 2023-05-11 at 9 36 28 AM" src="https://github.com/calcom/cal.com/assets/85004512/571b3d3c-4215-419f-ac6a-c9469652219b">
<img width="1319" alt="Screenshot 2023-05-11 at 9 36 16 AM" src="https://github.com/calcom/cal.com/assets/85004512/e5732ffb-110b-4cbc-a357-69a3238d4398">

Reply:
Thanks @aar2dee2. Looks like some weird problem. But as I see it, it has nothing to do with your app. Let's keep it reverted for this PR.
I would let @alannnc or @joeauyeung take care of it as they have more knowledge about this flow and this should be a problem with all video apps including Zoom in this case.

Reply:
okay. thanks!

any other recommended changes?